CMakeSettings.json

```diff
@@ -0,0 +1,45 @@
+{
+  // See https://go.microsoft.com//fwlink//?linkid=834763 for more information about this file.
+  "configurations": [
+    {
+      "name": "x64-Debug",
+      "generator": "Visual Studio 15 2017 Win64",
+      "configurationType": "Debug",
+      "inheritEnvironments": [ "msvc_x64_x64" ],
+      "buildRoot": "${thisFileDir}\\build\\${name}",
+      "cmakeCommandArgs": "",
+      "buildCommandArgs": "-v:minimal",
+      "ctestCommandArgs": "",
+      "variables": [
+        {
+          "name": "BOOST_ROOT",
+          "value": "C:\\lib\\boost_1_66_0"
```

Comment:
I don't love that we're hard coding the boost version here. Is there a better way? Maybe just `lib/boost`?

Reply:
I agree and that was part of my concern I wrote about in the first comment.  Perhaps we just recommend `C:\\lib\boost` as you say, and users can do some type of symlink if they want to manage multiple versions.